charts/selenium-grid/values.yaml

```diff
@@ -363,7 +364,7 @@ chromeNode:
     # Custom annotations for service
     annotations: {}
   # Size limit for DSH volume mounted in container (if not set, default is "1Gi")
-  dshmVolumeSizeLimit: 1Gi
+  dshmVolumeSizeLimit: 2Gi
```

Comment:
Why? This would affect people who are already using the chart, their clusters have been provisioned with the existing values.

Reply:
We had some issues while running it along with video recording which vanished as soon as we increased the limit. Think it should be part of PR. Will revert this.